File: ori_and_ter_finder_skew_graph.py
genome = "GCATACACTTCCCAGTAGGTACTG"
count = 0
loop_count = 0
print(count, end=" ")

for nucleotide in genome:
    if nucleotide == 'C':
        count -= 1
        print(count, end=" ")
    elif nucleotide == 'G':
        count += 1
        print(count, end=" ")
    else:
        print(count, end=" ")

    loop_count += 1
#Output a newline for better readability
print()

print(loop_count)

File: ori_and_ter_finder_skew_graph.cpp
#include<iostream>
#include<string.h>
using namespace std;
int main ()
{
    char genome[1000] = "TAAAGACTGCCGAGAGGCCAACACGAGTGCTAGAACGAGGGGCGTAAACGCGGGTCCGAT";
    int count = 0;
    int minima;

    cout<<count<<" ";
    
    for (int i=0; i<strlen(genome); i++)
    {
        if(genome[i] == 'C')
        {
            count--;
            cout<<count<<" ";
        }

        else if(genome[i] == 'G')
        {
            count++;
            cout<<count<<" ";
        }

        else 
        cout<<count<<" ";

    }
    return 0;
}